README.md

```diff
@@ -1,3 +1,166 @@
 # React Vis Network
 
 [![Build Status](https://travis-ci.org/lifeomic/react-vis-network.svg?branch=master)](https://travis-ci.org/lifeomic/react-vis-network)
+
+
+## Overview
+react-vis-network uses a declarative approach to defining [vis.js](https://github.com/almende/vis). networks. This allows developers to work in React using normal statefull approaches like [Redux](https://github.com/reduxjs/redux) and have the underlying network chart manage it's own lifecycle.
+
+## Example
+
+```js
+import React, { Component } from 'react';
+import { Network, Node, Edge } from 'react-vis-network';
+
+class MyNetwork extends Component {
+  render() {
+    return (
+      <Network>
+        <Node id='vadar' label='Darth Vadar' />
+        <Node id='luke' label='Luke Skywalker' />
+        <Node id='leia' label='Leia Organa' />
+        <Edge id='1' from='vadar' to='luke' />
+        <Edge id='1' from='vadar' to='leia' />
```

Comment:
Doesn't this Edge ID need to be unique from the vader->luke edge?

Reply:
_technically_, not any more, when Edge's had the code for decorators, if you used them, it would totally bust, but that just won't do anything anymore.